### ISSUE TITLE
USWDS-Site - 3.8.0 Documentation: Remove Date Picker calendar icon focus known issue

### ISSUE THREAD

**Issue by mahoneycm** [uswds/uswds-site#2506]
# Description

Remove resolved [known issue note](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-known-issues/components/date-picker/#known-issues-2) from Date Picker page. The issue was resolved in uswds/uswds#5701.

> The focus indicator is not visible around the calendar icon in high contrast mode. More information on GitHub: [uswds#5535](https://github.com/uswds/uswds/pull/5535)

**Comment by mahoneycm** [comment]
Closing in favor of https://github.com/uswds/uswds-site/pull/2483
